.github/workflows/kind-e2e.yaml

```diff
@@ -50,8 +50,7 @@ jobs:
           ${{ runner.os }}-go-
 
     # Install the latest release of ko
-    - name: Install ko
-      uses: ko-build/setup-ko@v0.6
```

Comment:
We want to keep `ko-build` - the imjasonh one is not being maintained

Reply:
done